enos/enos.py

```diff
@@ -40,13 +40,6 @@
 import pprint
 from operator import itemgetter, attrgetter
 
-from keystoneauth1.identity import v3
-from keystoneauth1 import session
-from glanceclient import client as gclient
-from keystoneclient.v3 import client as kclient
-from novaclient import client as nclient
-from neutronclient.neutron import client as ntnclient
-
```

Comment:
I'm sure there is a good reason to go with shell commands rather than python code. In this case, you should remove the dependency to the `request` package in the [setup.py](https://github.com/BeyondTheClouds/enos/blob/64628a6012bcdbb0e3a2ee1f8270dc188bbcc23d/setup.py#L25) and [enos.py](https://github.com/BeyondTheClouds/enos/blob/64628a6012bcdbb0e3a2ee1f8270dc188bbcc23d/enos/enos.py#L39)